src/main/default/lwc/caseAssistFlow/__tests__/caseAssistFlow.test.js

```diff
@@ -272,9 +286,7 @@ describe('c-case-assist-flow', () => {
       const testVisitorId = 'test-visitor-id';
       localStorage.setItem('visitorId', testVisitorId);
 
-      const testSubject = 'This is a test subject';
-
-      setSubject(element, testSubject);
+      setSubject(element, TEST_CASE.subject);
       // Flush microtasks
       await flushPromises();
```

Comment:
Why not rename the function `flushMicrotasks` instead and remove the comment?

Reply:
Promises are using microtasks internally to ensure callbacks are invoked in the proper order. At this level, we shouldn't have to care much about that. Let's simply remove the comment. :+1: